src/styles/globals.css

```diff
@@ -7,7 +7,7 @@
   --background-start-rgb: 214, 219, 220;
   --background-end-rgb: 255, 255, 255;
 
-  --card-size: 50vh;
+  --card-size: 500px;
```

Comment:
Did you change it to pixels for a reason? I remember before setting it to pixels fixed some stuff for me, but can we continue to test with 50vh, as that is how the current cards work so will need to be the same eventuallly...

Reply:
I set it to 500px cos we need to provide a width and height (in px) to Next/Image, so thought I'd match them. Just defaulting to 500px in the demo so we don't have to bother with `useNextSanityImage` or something to get attributes for the images.

That said it still works with `50vh` so I'll revert. But something to remember.